packages/boot/test/bootstrapTests/stack-linenumbers.test.ts

```diff
@@ -0,0 +1,42 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+/*
+lots of vertical space to test the display of error linenumbers
+when running TypeScript tests under Ava on Node.
+
+This is not an automated test of anything. Rather, its purpose is
+visual inspection of the output.
+
+With the `SUPPRESS_NODE_ERROR_TAMING` environment variable absent
+or set to `'disabled'`, you should see a stack trace that includes
+something like
+
+```
+  boot/test/bootstrapTests/stack-linenumbers.test.ts:1:104
+```
+
+This is because the TypeScript compiler compiles this file into one line
+of JavaScript with a sourceMap that should map back into original
+positions in this file. Node specifically makes use of that sourceMap
+to produce original linenumbers. However, Node does this in a way
+that resists virtualization, so the normal SES error taming cannot use
+this sourceMap info.
+
+However, if you also set  the `SUPPRESS_NODE_ERROR_TAMING` environment
+variable `'enabled'`, for example by doing
+
+```sh
+$ export SUPPRESS_NODE_ERROR_TAMING=enabled
+```
+at a bash shell, then when you run this test you should instead see
+something like
+```
+boot/test/bootstrapTests/stack-linenumbers.test.ts:40:32
+```
+
+*/
+
+test('foo', t => {
```

Comment:
```suggestion
test('for manual validation of error stack line number transparency', t => {
```

Reply:
the comment on this test is obsolete. and it's testing Endo so I don't think it should be included in agoric-sdk.

so I'd request that it be removed before merging. alternately, since this PR looks ready to merge, it could be removed as part of https://github.com/Agoric/agoric-sdk/pull/9735

Or since that may be imminent we could close this PR and wait for that

Reply:
the comment on this test is obsolete. and it's testing Endo so I don't think it should be included in agoric-sdk.

so I'd request that it be removed before merging. alternately, since this PR looks ready to merge, it could be removed as part of https://github.com/Agoric/agoric-sdk/pull/9735

Or since that may be imminent we could close this PR and wait for that

Reply:
the comment on this test is obsolete. and it's testing Endo so I don't think it should be included in agoric-sdk.

so I'd request that it be removed before merging. alternately, since this PR looks ready to merge, it could be removed as part of https://github.com/Agoric/agoric-sdk/pull/9735

Or since that may be imminent we could close this PR and wait for that

Reply:
Done

Reply:
Hi @turadg , fortunately I saw your comment in time. I'll remove. Thanks.

Reply:
Done